react-main/src/pages/SacDm/components/SacDmDevice.tsx

```diff
@@ -4,23 +4,33 @@ import { EmptyData } from "../../../components/EmptyData";
 import React, { useCallback, useEffect, useState } from "react";
 import { SacDmDefaultProps } from "../../../types";
 import sacDmDefault from "../../../app/services/sacdm_default";
+import styled from "styled-components";
+
+const Divider = styled.hr`
+  border: none;
+  border-top: 2px solid ${({ theme }) => theme.gray800};
+  margin: 20px 0;
+`;
+
+const Section = styled.div`
+  margin-bottom: 40px;
+`;
 
 export const SacDmDevice = ({
   deviceId,
   sacDm,
 }: {
-  deviceId: number | null;
+  deviceId: number ;
   sacDm: SacDmProps[];
 }) => {
   const [sacDmMean, setsacDmMean] = useState<SacDmDefaultProps>();
 
   const loadSacDmDefault = useCallback(async () => {
     try {
-      const response = await sacDmDefault.getSacDmDefault(1);
-      console.log("Response: ", response);
+      const response = await sacDmDefault.getSacDmDefault(deviceId);
+      setsacDmMean(response);
       
       // TODO: Alterar para pegar o veículo selecionado
```

Comment:
Esse comentário ainda faz sentido? Ainda está como To Do ou você implementou com essa sua alteração?